### PR TITLE
C368482 Browse and Search capabilities are separated (spitfire) (TaaS)

### PR DESCRIPTION
**Pull request by manowarqqq**
https://foliotest.testrail.io/index.php?/cases/view/368482
https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/5011/allure/#suites/70cae361de89575d08e2aa2ffb0cebf1/e098057af5d34cf1/
image

### CODE REVIEW

File: cypress/e2e/inventory/search/browse-and-search-capabilities-are-separated.cy.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+const contributorsNameTypes = ['Personal name', 'Corporate name', 'Meeting name'];
+const Dropdowns = {
+  EFFECTIVE_LOCATION: 'Effective location (item)',
+  NAME_TYPE: 'Name type',
+};
+describe('Inventory', () => {
+  describe('Search in Inventory', () => {
+    before('Create test data', () => {
+      cy.getAdminToken();
+      cy.createTempUser([Permissions.uiSubjectBrowse.gui]).then((userProperties) => {
```
Comment (ernestyedigaryan1) [inline cypress/e2e/inventory/search/browse-and-search-capabilities-are-separated.cy.js:22]: 2 more permissions are missing, please add them also

**Review by ernestyedigaryan1 — APPROVED** [review]

**Review by Sherzod-Kenjaev — COMMENTED** [review]
Add two more missing user permissions please

File: cypress/e2e/inventory/search/browse-and-search-capabilities-are-separated.cy.js (hunk 1 above)
Comment (manowarqqq) [reply cypress/e2e/inventory/search/browse-and-search-capabilities-are-separated.cy.js]: done

**Comment by manowarqqq** [comment]
> Add two more missing user permissions please

done

**Review by Sherzod-Kenjaev — APPROVED** [review]

**Pull request merged**